internal/testclient/testtx/context.go

```diff
@@ -23,32 +23,39 @@ import (
 	"pocket/pkg/client/tx"
 )
 
+// NewLocalnetContext creates and returns a new transaction context configured
+// for use with the localnet sequencer.
+func NewLocalnetContext(t *testing.T) client.TxContext {
+	t.Helper()
+
+	flagSet := testclient.NewLocalnetFlagSet(t)
+	clientCtx := testclient.NewLocalnetClientCtx(t, flagSet)
+	txFactory, err := cosmostx.NewFactoryCLI(*clientCtx, flagSet)
+	require.NoError(t, err)
+	require.NotEmpty(t, txFactory)
+
+	deps := depinject.Supply(
+		*clientCtx,
+		txFactory,
+	)
+
+	txCtx, err := tx.NewTxContext(deps)
+	require.NoError(t, err)
+
+	return txCtx
+}
+
 // TODO_IMPROVE: these mock constructor helpers could include parameters for the
 // "times" (e.g. exact, min, max) values which are passed to their respective
 // gomock.EXPECT() method calls (i.e. Times(), MinTimes(), MaxTimes()).
 // When implementing such a pattern, be careful about making assumptions about
 // correlations between these "times" values and the contexts in which the expected
 // methods may be called.
 
-// NewOneTimeErrTxTimeoutTxContext creates a mock transaction context designed to simulate a specific
```

Comment:
Changes from here down were #94. I'm not sure why github is showing this diff  here.